docs/Concepts/Client-Libraries.md

```diff
@@ -0,0 +1,13 @@
+description: Hyperledger Besu client libraries 
+<!--- END of page meta data -->
+
+# Client Libraries
+
+Client libraries such as web3.js, web3j, or ethereumj are used by Dapps to forward JSON-RPC requests to Hyperledger Besu. Any
+client library that implements core Ethereum RPC methods can be used with Besu.
+
+The web3.js-eea library can be used with Besu for [privacy features](../Concepts/Privacy/Privacy-Overview.md).
+
+![Client Libraries](../images/Hyperledger-Besu-Client-Libraries.png)
+
+Client libraries can also be used to [create signed transactions](../HowTo/Develop-Dapps/Use-web3js.md). Use the web3.js-eea library to [create and send private transactions](../HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md).
```

Comment:
Would this be better as a bulleted list? eg, 

Use client libraries to: 
* Create signed transactions 
* Create and send private transactions

Reply:
I was two-minded about this. I'll make the change since you also recommending this.